packages/common/tests/test-aws.js

```diff
@@ -149,3 +149,36 @@ test('pullStepFunctionEvent returns message from S3', async (t) => {
     stub.restore();
   }
 });
+
+const throwThrottlingException = () => {
```

Comment:
Should this use the functions you've defined in `common/test-utils.js`?

Reply:
Good catch, yes.  I'll fix that.

Reply:
Fixed.